lumen/ai/agents.py

```diff
@@ -739,34 +787,18 @@ async def answer(self, messages: list | str):
 
         tables_sql_schemas = {
             table: {
-                "schema": self._get_schema(source, table, include_min_max=False),
+                "schema": self._get_schema(source, table),
```

Comment:
Decided the min/max is still useful for the LLM to know the format of the values.